tools/copy_production_db_to_staging

```diff
@@ -0,0 +1,162 @@
+        required "$STAGING_RDS_INSTANCE_TYPE" "Instance class"
+
+        STAGING_RDS_SUBNET_GROUP="$(aws rds describe-db-instances \
+            --db-instance-identifier $STAGING_ID \
+            --query "DBInstances[0].DBSubnetGroup.DBSubnetGroupName" \
+            --output text)"
+
+        STAGING_RDS_VPC_SECURITY_GROUP="$(aws rds describe-db-instances \
+            --db-instance-identifier $STAGING_ID \
+            --query "DBInstances[0].VpcSecurityGroups[0].VpcSecurityGroupId" \
+            --output text)"
+
+        # Remove rds: from snapshot ID & use as database ID
+        NEW_DATABASE_ID="${SNAPSHOT_ID:4}"
+
+        RESTORE="aws rds restore-db-instance-from-db-snapshot \
+            --no-paginate \
+            --db-instance-identifier $NEW_DATABASE_ID \
+            --db-snapshot-identifier $SNAPSHOT_ID \
+            --db-instance-class $STAGING_RDS_INSTANCE_TYPE \
+            --db-subnet-group-name $STAGING_RDS_SUBNET_GROUP\
+            --vpc-security-group-ids $STAGING_RDS_VPC_SECURITY_GROUP"
+
+        echo "Will run '$RESTORE'"
+        read -n 1 -s -r -p "Press any key to begin"
+        echo
+
+        $RESTORE
+
+        # Restore will complete right away, but we can't begin to query
+        # the database until the status is "available"
+        CHECK_STATUS="aws rds describe-db-instances \
+            --db-instance-identifier $NEW_DATABASE_ID \
+            --query DBInstances[0].DBInstanceStatus --output text"
+
+        echo -n "Polling until new instance is ready"
+        while [ "$($CHECK_STATUS)" != 'available' ]; do
+            echo -n "."
+            sleep 10
+        done
+        echo
+
+        NEW_DATABASE_ENDPOINT="$(aws rds describe-db-instances \
+            --db-instance-identifier $NEW_DATABASE_ID \
+            --query DBInstances[0].Endpoint.Address --output text)"
+
+        pushd "$PROJECT_ROOT/load-tests/anonymize-the-database"
+
+        echo "Installing requirements for anonymization script"
+        python3 -m venv ./venv
+        source ./venv/bin/activate
+        pip install -r requirements.txt
+
+        ANONYMIZATION_QUERIES="$(PGPASSWORD=$PRODUCTION_PGPASSWORD PGHOST=$NEW_DATABASE_ENDPOINT \
+                                 PGPORT=5432 ALLOWED_EMAIL=$ALLOWED_EMAILS \
+                                 ./generate_anon_queries)"
+
+        popd
+
+        read -n 1 -s -r -p "Press any key to start anonymizing the database"
+        echo
+        export PGPASSWORD="$PRODUCTION_PGPASSWORD" 
+        echo "$ANONYMIZATION_QUERIES" | psql -h "$NEW_DATABASE_ENDPOINT" -U openapparelregistry -t -A openapparelregistry -f -
+
+        # Run pg_dump
+        read -n 1 -s -r -p "Press any key to start pg_dump"
+        echo
+        DUMP_FILE="$NEW_DATABASE_ID.dump"
+        pg_dump -Fc -h "$NEW_DATABASE_ENDPOINT" -U openapparelregistry openapparelregistry > "$DUMP_FILE"
+
+        # Upload dump to S3
+        aws s3 cp "$DUMP_FILE" "s3://$S3_BUCKET/$DUMP_FILE"
```

Comment:
I had written this copy to s3 into the requirements based on a previous workflow and now realize that it is not needed. To avoid piling up dumps in the bucket I recommend removing this cp step